frontend/src/scenes/events/definitions/DefinitionDrawer.tsx

```diff
@@ -1,6 +1,6 @@
 import { useActions, useValues } from 'kea'
```

Comment:
This file desperately wants to be split into multiple small files. 

Golden rule: one exported component per file.